src/components/files-grid-viewer.less

```diff
@@ -0,0 +1,42 @@
+@files-grid-viewer-file-icon-padding: 10px;
+@files-grid-viewer-file-container-base-width: 60px;
+@files-grid-viewer-file-container-total-width: @files-grid-viewer-file-container-base-width + 2 * @files-grid-viewer-file-icon-padding;
+
+.FilesGridViewer {
+  display: flex;
+  gap: 20px;
+  flex-wrap: wrap;
+  flex-direction: row;
+  justify-content: flex-start;
+  padding: @edu-content-padding;
+  border: 1px solid @edu-border-color-base;
+}
+
+.FilesGridViewer-fileContainer {
+  width: @files-grid-viewer-file-container-total-width;
+  overflow: hidden;
+  white-space: nowrap;
+  text-decoration: none;
+  text-overflow: ellipsis;
+  color: @edu-text-color;
+
+  &:hover,
+  &:active {
+    text-decoration: none;
+    color: @edu-text-color;
+  }
+
+  &:hover {
+    font-weight: bold;
+
+    .FilesGridViewer-fileIcon {
+      font-size: 65px;
+      padding-top: 5px;
+    }
+  }
+}
+
+.FilesGridViewer-fileIcon {
+  font-size: 60px;
```

Comment:
We could put a `transition: all .2s ease-in-out;` here in order to make the hover effect less "jumpy".